### ISSUE TITLE
AWS Elasticsearch API

### ISSUE THREAD

**Issue by minitomy** [fog/fog-aws#286]
Hello,
I want to make use of the AWS Elasticsearch API to read the domain status, just like the one available for Elasticache - describe_clusters. However, in fog-aws, I could not find the same.
Could anyone guide me as to how can I go about accessing the same?

Best regards,
Mini

**Comment by lanej** [comment]
I believe you are referring to [DescribeCacheClusters](http://docs.aws.amazon.com/AmazonElastiCache/latest/APIReference/API_DescribeCacheClusters.html) which is available via [#describe_cache_clusters](https://github.com/fog/fog-aws/blob/master/lib/fog/aws/requests/elasticache/describe_cache_clusters.rb)

**Comment by minitomy** [comment]
I understand how to get Elasticache. But I need to fetch details of the ElasticSearch. How do I get them?

**Comment by lanej** [comment]
@minitomy it appears we do not currently have support for Elasticsearch. (Confirm @geemus ?).

**Comment by geemus** [comment]
@minitomy @lanej Yeah, I don't think we have had anybody ask before (that I recall). We can certainly support an effort to support it, but most things like this get added in a situation like this (somebody needs it and has some time/energy to make it happen).

**Comment by geemus** [comment]
Sounds good, thanks for the update.
